app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for person in people:
+
+        Person(name=person["name"], age=person["age"])
+
+    for person in people:
+
+        name = person["name"]
+
+        if "wife" in person and person["wife"]:
+            Person.people[name].wife = Person.people[person["wife"]]
+
+        if "husband" in person and person["husband"]:
+            Person.people[name].husband = Person.people[person["husband"]]
+
+    return [person for person in Person.people.values()]
```

Comment:
```suggestion
    for person in people:
        Person(name=person["name"], age=person["age"])

    for person in people:
        name = person["name"]

        if "wife" in person and person["wife"]:
            Person.people[name].wife = Person.people[person["wife"]]

        if "husband" in person and person["husband"]:
            Person.people[name].husband = Person.people[person["husband"]]

    return list(Person.people.values())
```

Reply:
Excuse me, now I have to add your suggestions to the commit?!?